samples/web/content/apprtc/apprtc.py

```diff
@@ -312,14 +318,25 @@ def __str__(self):
 class Room:
   def __init__(self):
     self.clients = {}
+    self.allowed_clients = None;
```

Comment:
It needs some documentation:
1. the initiator is also included in the list
2. anyone is allowed if it's None